### DISCUSSION TITLE
No documentation / explanation on how to set a background image

### DISCUSSION THREAD

**Discussion by LainIwakura** [Eugeny/tabby#2745] [General]
I love terminus but the one thing I'd love more is the ability to set a background image. From what I gather, this can be done through the custom CSS but it doesn't tell you which element to select and the only search results on this topic are a few years old.

Can the documentation please be updated so it indicates which elements in the app need to be referenced for a background to be appropriately set? Ideally the background should 'blur' with the colour and retain some opacity. Other terminals like iTerm2 do support this feature.

**Comment by washicjr** [comment]
CSS documentation would be wonderful. Hacking about trying to find the right CSS element to do simple things like increase margins is a pain.

**Comment by Eugeny** [comment]
Hint: you can open the DevTools from Settings > Application, which gives you an element tree and style editor.

**Reply by TzeroOcne** [reply]
I wish something like this available in the website so I can find it easily, it's kinda a hassle to go to github discussion to find this, I know I should inspect the settings more but still
